runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -3170,11 +3168,10 @@ J9::Z::PrivateLinkage::storeExtraEnvRegForBuildArgs(TR::Node * callNode, TR::Lin
      }
    return stackOffset;
    }
-
-////////////////////////////////////////////////////////////////////////////////
+/////////////////////////////////////////////////////////////////////////////
```

Comment:
It looks like a blank line was removed and the number of slashes on this line changed.  It should look like this:

```
   }

////////////////////////////////////////////////////////////////////////////////
// J9::Z::PrivateLinkage::addFECustomizedReturnRegDependency -  add extra
// linkage specific return register dependency
```